File: src/gestorAplicacion/clasesPrincipales/Servicio.java
package gestorAplicacion.clasesPrincipales;

public enum Servicio {
	
	MALETAS_EXTRA(5000),
	ALMOHADA(2500),
	AURICULARES(3000),
	VIAJAR_CON_MASCOTA(10000);
	
	private final int precio;
	
	Servicio(int precio) {
		this.precio=precio;
	}

	public int getPrecio() {
		return precio;
	}
	
	
}

File: Python/gestor_aplicacion/clasesPrincipales/Servicio.py
from enum import Enum

class Servicio(Enum):
    MALETAS_EXTRA = 5000
    ALMOHADA = 2500
    AURICULARES = 3000
    VIAJAR_CON_MASCOTA = 10000

    def getPrecio(cls):
        return Servicio.value

    
